src/react/projects/spark-react/src/SprkMasthead/SprkMasthead.js

```diff
@@ -78,33 +78,33 @@ class SprkMasthead extends Component {
         data-id={idString}
       >
         <div className="sprk-c-Masthead__content sprk-o-Stack__item sprk-o-Stack sprk-o-Stack--split@xxs">
-          <SprkMastheadMenuIcon toggleNarrowNav={this.toggleNarrowNav} isOpen={narrowNavOpen} />
+          <SprkMastheadMenuIcon
+            toggleNarrowNav={this.toggleNarrowNav}
+            isOpen={narrowNavOpen}
+          />
 
           <div className="sprk-c-Masthead__branding sprk-o-Stack__item sprk-o-Stack__item--center-column@xxs">
-            <a href="/">
+            <SprkLink variant="unstyled" href="/">
```

Comment:
This isn't related to this link issue. I see that href here is set to "/" and it looks like its wrapping the site Logo. I think we might want to provide a prop for the site logo href and also, it's possible that they would want to choose if this should be an internal link or external. So we should also give the option of "element" for this. Is this something you want a new issue for? @guern1kn

Reply:
yeah, I think because we will have additional unit tests to write this should be its own issue. I'll talk to Steph.